src/frontend/komponenter/Fagsak/Brevmottaker/LeggTilEndreBrevmottakerModal.tsx

```diff
@@ -109,122 +117,133 @@ export const LeggTilEndreBrevmottakerModal: React.FC = () => {
         adresseKilde === AdresseKilde.OPPSLAG_ORGANISASJONSREGISTER;
 
     return (
-        <StyledModal
-            open={visBrevmottakerModal}
-            aria-label={heading}
-            onClose={lukkModal}
-            shouldCloseOnOverlayClick={false}
-        >
-            <StyledHeading level="2" size="medium" id="modal-heading">
-                {heading}
-            </StyledHeading>
-            <FlexContainer>
-                <StyledFieldset
-                    legend="Skjema for å legge til eller redigere brevmottaker"
-                    hideLegend
-                    error={skjema.visFeilmeldinger && hentFrontendFeilmelding(skjema.submitRessurs)}
-                >
-                    <MottakerSelect
-                        {...skjema.felter.mottaker.hentNavBaseSkjemaProps(skjema.visFeilmeldinger)}
-                        label="Mottaker"
-                        onChange={(event): void => {
-                            const nyMottakerType = event.target.value as MottakerType;
-                            if (skalNullstilleAdresseInputVedNyMottakerType(nyMottakerType)) {
-                                nullstillManuellAdresseInput();
-                                settAdresseKilde(
-                                    erMottakerBruker(nyMottakerType)
-                                        ? AdresseKilde.MANUELL_REGISTRERING
-                                        : AdresseKilde.UDEFINERT
-                                );
-                            }
-                            skjema.felter.mottaker.validerOgSettFelt(nyMottakerType);
-                        }}
+        <StyledModal open={visBrevmottakerModal} aria-label={heading} onBeforeClose={lukkModal}>
+            <StyledModalHeader closeButton={true}>
+                <StyledHeading level="2" size="medium" id="modal-heading">
+                    {heading}
+                </StyledHeading>
+            </StyledModalHeader>
```

Comment:
Her tror jeg vi kan bruke header-prop i Modalen i stedet for å sette header eksplisitt på denne måten. Da trenger du heller ikke å sette aria-label, fordi den fikser det for deg automatisk

Se eksempel fra ba-sak-frontend:
```
<Modal
                    open
                    width={'35rem'}
                    onClose={() => {
                        nullstillSkjema();
                        nullstillDokument();
                        settVisModal(false);
                    }}
                    header={{ heading: 'Henlegg behandling', size: 'medium' }}
                >
```

Reply:
Det var for å bevare eksisterende formattering - men ser poenget ditt 😊 Oppdatert.